frontend/test/features/withdraw/withdraw_page_test.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter_starter/features/withdraw/withdraw_page.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/widget_helpers.dart';
+
+void main() {
+  group('WithdrawPage', () {
+    testWidgets('should show withdraw input and output amounts',
+        (tester) async {
+      await tester.pumpWidget(
+        WidgetHelpers.testableWidget(child: const WithdrawPage()),
+      );
+      final depositAmountPattern = RegExp(r'\$[0-9]+\.[0-9]{2}$');
+
+      expect(find.textContaining(depositAmountPattern), findsExactly(2));
```

Comment:
Will these amounts not be consistent in these tests? Like can we `find.text` with exact matches? Or do we need the regex here.